src/components/admin/applications/AllApplicationsTable.tsx

```diff
@@ -40,15 +47,30 @@ const AllApplicationsTable: React.FC = () => {
   const { hexathonId } = useParams();
   const [offset, setOffset] = useState(0);
   const [searchText, setSearchText] = useState("");
+
+  const [status, setStatus] = useState<string[]>([]);
+  const [applicationBranch, setApplicationBranch] = useState<string[]>([]);
+  const [confirmationBranch, setConfirmationBranch] = useState<string[]>([]);
+
   const [{ data, error }] = useAxios({
     method: "GET",
     url: apiUrl(Service.REGISTRATION, "/applications"),
     params: {
       hexathon: hexathonId,
+      status,
+      applicationBranch,
+      confirmationBranch,
       search: searchText,
       offset,
     },
   });
+  const [{ data: branches, error: error2 }] = useAxios({
```

Comment:
instead of `error2`, could u say something more descriptive like `branchesError`?